topics/cordova/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: apache-cordova
+created_by: Apache Software Foundation
+display_name: Cordova
+github_url: https://github.com/apache?utf8=%E2%9C%93&q=cordova-
```

Comment:
seems like this might be a better link?
```suggestion
github_url: https://github.com/apache/cordova
```

Reply:
Right 😃 I followed the _Source Code_ link from Cordova homepage, but this one is best suited